docs/installation/local.mdx

```diff
@@ -64,8 +128,30 @@ Configure Capact CLI to connect to your fresh local Capact installation. Follow
 
 To remove your local Kubernetes cluster created in the [Create local Kubernetes cluster](#create-local-kubernetes-cluster) section, run the following command:
 
+
+<Tabs
+groupId="ref"
+defaultValue="kind"
+values={[
+{label: 'Kind', value: 'kind'},
+{label: 'K3d', value: 'k3d'},
+]}>
+<TabItem value="kind">
+
 ```bash
 capact environment delete kind
 ```
 
-> **NOTE:** This command deletes the local cluster with all data, including history of executed Actions and created TypeInstances. Make sure you cleaned up all external resources (e.g. managed PostgreSQL databases) based on the TypeInstance data.
+</TabItem>
+<TabItem value="k3d">
+
+```bash
+capact environment delete k3d
+```
+
+</TabItem>
+</Tabs>
+
+:::note
+This command deletes the local cluster with all data, including history of executed Actions and created TypeInstances. Make sure you cleaned up all external resources (e.g. managed PostgreSQL databases) based on the TypeInstance data.
+:::
```

Comment:
I would move this note above the tabs, to avoid the situation someone runs one of the commands, and then reads the note 😄 
And also I would recommend using `danger`:

 ```suggestion
:::danger
This command deletes the local cluster with all data, including history of executed Actions and created TypeInstances. Before running it, make sure you cleaned up all external resources (e.g. managed PostgreSQL databases) based on the TypeInstance data.
:::
```